test/provision/k8s_install.sh

```diff
@@ -147,6 +174,14 @@ case $K8S_VERSION in
         sudo ln -sf $COREDNS_DEPLOYMENT $DNS_DEPLOYMENT
         KUBEADM_CONFIG="${KUBEADM_CONFIG_ALPHA2}"
         ;;
+    "1.13")
+        KUBERNETES_CNI_VERSION="0.6.0"
+        K8S_FULL_VERSION="1.13.0-beta.2"
+        KUBEADM_OPTIONS="--ignore-preflight-errors=cri"
+        KUBEADM_SLAVE_OPTIONS="--discovery-token-unsafe-skip-ca-verification --ignore-preflight-errors=cri,SystemVerification"
+        sudo ln -sf $COREDNS_DEPLOYMENT $DNS_DEPLOYMENT
```

Comment:
I think that we should be consistent here, if we run a different version of coredns for 1.12 we also need to run the right version for 1.13. I strongly disagree with this one (We should also use the latest), but if you can add the coredns version for 1.13 will be great.